test/runtime/specs/juttle-spec/flowgraph-tests.spec.md

```diff
@@ -14,3 +14,21 @@ Modifying a point on one branch does not modify it on another.
     {"label":"branch1"}
     {"label":"branch1"}
     {"label":"pre"}
+
+Pints with `time == null` are handled correctly
+-----------------------------------------------
+
+Regression test for #351.
+
+### Juttle
+
+    emit -from :0: -limit 1
+    | batch -every :1s:
+    | filter true = false
+    | (reduce time = null; reduce time = null)
```

Comment:
I strive for the test cases to be minimal so that understanding them & debugging any failure is as painless as it can be. So I’ll keep the test case as it is. I think the reference to #351 above is enough of a pointer to the original problem.